docs/DEPLOYMENT.md

```
@@ -0,0 +1,505 @@
+    stats = geocoder.get_stats()
+    print(f'Success rate: {stats.get(\"success_rate\", 0):.1%}')
+    print(f'Cache hit rate: {stats.get(\"cache_hit_rate\", 0):.1%}')
+
+asyncio.run(check_stats())
+"
+```
+
+### Log Management
+
+Configure log rotation in `/etc/logrotate.d/dealgenie`:
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)